src/request_rate_worker.h

```diff
@@ -115,6 +116,8 @@ class RequestRateWorker : public LoadWorker, public IScheduler {
     ctx->SetNumActiveThreads(num_threads_);
   }
 
+  const std::chrono::milliseconds delay_tolerance_{1};
```

Comment:
This could eventually be a user option, but I want to keep it fixed until someone asks otherwise.